transform/transformers/PCKTransformer.py

```diff
@@ -40,7 +47,19 @@ def get_form_questions(self):
     def get_cs_form_id(self):
         instrument_id = self.response['collection']['instrument_id']
 
-        return self.form_ids[instrument_id]
+        try:
+            form_type = self.form_types[self.survey['survey_id']]
+        except KeyError:
+            logger.error("Invalid survey id '{}'".format(self.survey['survey_id']))
+            return 'XXXXXX'
```

Comment:
Do the clients of this method know what to do with the XX's or will they attempt to treat them as a valid form id?